catalog/catalog.go

```diff
@@ -0,0 +1,266 @@
+package catalog
+
+import (
+	"archive/tar"
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"github.com/banzaicloud/pipeline/config"
+	"github.com/banzaicloud/pipeline/helm"
+	"github.com/pkg/errors"
+	"gopkg.in/yaml.v2"
+	"io"
+	helm_env "k8s.io/helm/pkg/helm/environment"
+	"k8s.io/helm/pkg/repo"
+	"k8s.io/helm/pkg/strvals"
+	"regexp"
+	"strings"
+)
+
+const CatalogRepository = "catalog"
+const CatalogRepositoryUrl = "http://kubernetes-charts.banzaicloud.com/branch/spotguide"
+
+var CatalogPath = "./" + CatalogRepository
+
+//TODO when the API fixed this needs to move to banzai-types
```

Comment:
Is this fixed?

Reply:
It won't until the UI is fully functional